core/src/main/resources/db/migration.sql

```diff
@@ -270,3 +270,7 @@ ALTER TABLE `genetic_entity` DROP COLUMN `TMP_GENE_ID`;
 -- ========================== end of genetic_entity related migration =============================================
 
 UPDATE info SET DB_SCHEMA_VERSION="2.0.0";
```

Comment:
because every migration in this file is independent from each other, so every block should end with setting db schema to respective version.